content/en/docs/concepts/extend-kubernetes/api-extension/_index.md

```diff
@@ -1,4 +1,14 @@
 ---
 title: Extending the Kubernetes API
 weight: 20
+content_type: concept
 ---
+
+
+<!-- Overview -->
+
+A _Custom Resource Definition_ (CRD) defines the name and structure of a custom object type that you want to allow in your Kubernetes cluster. A CRD is a way of defining what the objects will be called along with properties it can possess like _size, color_ etc.
+
+_Kubernetes Aggregation Layer_ allows Kubernetes to be extended with additional APIs, beyond what is offered by the core Kubernetes APIs. The additional APIs can either be ready-made solutions such as a metrics server, or APIs that you develop yourself.
+
+_To have a better understanding about the topic, It is recommended to go through the below links_.
```

Comment:
@NitishKumar06 This PR needs significant refactoring and should not merge in its current form. I talked with @reylejano and he'll review it more thoroughly on his shift next week.

In the meantime, to echo the advice you've received from @divya-mohan0209 and others in different contexts, I recommend that you carefully review the style guide, especially the section about using [simple and direct language](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language).

Reply:
@zacharysarah Thanks for sharing! I just read it now and found that there are errors I'm committing. I'll make the changes again keeping in mind the styling guide of Kubernetes.

Reply:
@NitishKumar06 I opened a similar PR that adds a section overview. Have a look at https://deploy-preview-34734--kubernetes-io-main-staging.netlify.app/docs/concepts/extend-kubernetes/compute-storage-net/ and compare it to the [original](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/) autogenerated page.

Maybe that'll be useful as a reference?

Reply:
Also, have a look at https://deploy-preview-34734--kubernetes-io-main-staging.netlify.app/docs/concepts/extend-kubernetes/#api-extensions (from the same PR) for some more hints on what to explain.